sdk/communication/Azure.Communication.PhoneNumbers/src/Azure.Communication.PhoneNumbers.csproj

```diff
@@ -18,8 +18,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- <PackageReference Include="Azure.Core" /> -->
-    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
+    <PackageReference Include="Azure.Core" />
+    <!-- <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" /> -->
```

Comment:
Please remove this commented line, it doesn't add any value to us other than making this even more confusing.